src/Collection.php

```diff
@@ -166,9 +166,9 @@ public function append(...$items): CollectionInterface
         return new self(Append::of()(...$items), $this->getIterator());
     }
 
-    public function apply(callable ...$callables): CollectionInterface
+    public function apply(callable ...$callbacks): CollectionInterface
```

Comment:
I think we should normalize this everywhere.

Let's leave it like this today, but this is something I would like to change and normalize everywhere.